test/e2e/placement_test.go

```diff
@@ -44,6 +46,18 @@ var _ = ginkgo.Describe("Placement", func() {
 	})
 
 	ginkgo.AfterEach(func() {
+		ginkgo.By("Delete managedclusterset")
+		err = clusterClient.ClusterV1alpha1().ManagedClusterSets().Delete(context.Background(), clusterSet1Name, metav1.DeleteOptions{})
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+		ginkgo.By("Delete managedclusters")
+		for _, mcName := range managedClusterNames {
+			if len(mcName) > 0 {
+				err = clusterClient.ClusterV1().ManagedClusters().Delete(context.Background(), mcName, metav1.DeleteOptions{})
```

Comment:
I think we can use DeleteCollection here to delete all managedclusters or adding a special labels for e2e test?

Reply:
Great idea.
Updated the codes with DeleteCollection, it is very clean now.